docs/docs/50-roadmap.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Roadmap
+Description: Kargo Roadmap
+---
+
+# Roadmap
+
+Kargo does not currently have a regular release cadance. The plan is to do a few releases, and based on the resluts, create a more formal release cadance.
+
+:::caution
+This roadmap is subject to change at any time, for the most up to date information, please see the [GitHub Project](https://github.com/akuity/kargo/projects)
+:::
+
+# v0.2.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `Freight/Warehouse CRD` | feature | Freight will change from being a property of a `Stage`, to being it's own CRD. A collection of promotable `Freight` will be known as a `Warehouse`. |
+| `Long Lived Promotions` | feature | Support for Pull Requset based promotion, which is indefinite. This will allow the promotion to last as long as the PR is open. |
```

Comment:
Instead of "Long Lived Promotions", GitHub PR-based promotion is the manifestation of a long-lived promotion that folks will understand better.